common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -34,6 +34,16 @@ trait ABTestSwitches {
     exposeClientSide = true,
   )
 
+  Switch(
+    ABTests,
+    "ab-sign-in-gate-mandatory",
```

Comment:
To confirm, is this supposed to be the JavaScript test ID with camel case replaced with dashes and prefixed with `ab-`?

Reply:
yeah that's right